### PR TITLE
Added my advice link 

### PR DESCRIPTION
**Pull request by SophieADav**
Added a link to my "Advice" page on the index.html.

### CODE REVIEW

**Comment by twosavoie** [comment]
Hi @SophieADav,

Fantastic work! This is great!

One small change to make before we can merge in your code. There are a few merge conflict markers in the index.html file. (You can see them [here](https://github.com/skillcrush/git-practice/pull/1085/files).) If you could remove them while on the advice branch, then add and commit the changes and push to GitHub, you pull request will be updated automatically. 

Let us know if you have any questions!

Best,
Lisa

**Comment by dgre87** [comment]
12166@DESKTOP-CSHVQ4P MINGW64 ~/git-practice (advice)
$ git push origin advice
Enter passphrase for key '/c/Users/121
66/.ssh/id_rsa':
Enumerating objects: 19, done.
Counting objects: 100% (19/19), done.
Delta compression using up to 4 thread
s
Compressing objects: 100% (13/13), don
e.
Writing objects: 100% (13/13), 2.69 Ki
B | 343.00 KiB/s, done.
Total 13 (delta 6), reused 0 (delta 0)
, pack-reused 0
remote: Resolving deltas: 100% (6/6),
completed with 2 local objects.
To github.com:dgre87/git-practice.git
   62006de..1da5e67  advice -> advice

12166@DESKTOP-CSHVQ4P MINGW64 ~/git-practice (advice)
$

**Comment by twosavoie** [comment]
Hi @SophieADav,

It looks like you made a push to GitHub, but we still see the conflict markers in the index.html file. Once you've removed them, can you make sure to save your work in your text editor? Then add the changes and commit as usual. Once you push to GitHub, your pull request should be updated. 

If you have any questions or need help with this, let us know! 🙂 

Best,
Lisa
Screenshot 2023-07-11 at 11 36 20 PM (2)

**Comment by SophieADav** [comment]
Hi Lisa,

I can't see any conflicts on my Text Editor or on the Terminal app when I
try to merge them. Do I need to take it from GitHub again to fix it or am I
missing something? It didn't flag up when I tried to merge them on Terminal
as it had before so I'm not too sure. Thanks for any help!

Thanks,
Sophie

**Comment by twosavoie** [comment]
Hi @SophieADav,

Can you check to be sure you're on the advice branch and then look in the index.html file at about lines 1254 - 1264? When I look at your GitHub repo on the advice branch, I see [conflict markers in the index.html file](https://github.com/SophieADav/git-practice/blob/cf23c795bb3647a0cef71db788181a2bff48c449/index.html#L1254). 

If you don't see the same locally, perhaps you already removed them? Could you run **git status** to see if there are any changes to be added and committed? And then could you push the advice file to GitHub. It will update your pull request automatically. 

If you're unsure, send us a screenshot of your index.html file in your text editor at lines 1254 - 1264 while on the advice branch. And a screenshot of your command line where you ran **git status**. We'll be able to go from there. 

Best,
Lisa

**Comment by twosavoie** [comment]
Hi @SophieADav,

Just in case it's helpful, I thought I would reach out and let you know that there's a [Live Q&A](https://learn.skillcrush.com/events/) tomorrow (Thurs. US) where you could meet with Ann and she could answer any questions about your pull request. 

Best,
Lisa

**Comment by SophieADav** [comment]
Hi Lisa,

Thanks for all your help! I think I fixed it now, here's a link to the
repo: https://github.com/SophieADav/git-practice/blob/main/index.html.

Thanks,
Sophie

File: index.html [hunk 1]
```diff
@@ -1251,12 +1251,8 @@ <h2>Our Advice for Newbies!</h2>
       <a href="advice/lyla-advice.html">Lyla's Advice</a><br>
       <a href="advice/SueWigand-advice.html">Sue Wigand's Advice</a><br>
       <a href="advice/jeanette-h-advice.html">Jeanette H's Advice</a><br>
```
Comment (twosavoie) [inline index.html:1253]: The six links below have been removed from the index.html file. Once you've added them back you can add and commit the change and then push your code to GitHub.

**Comment by twosavoie** [comment]
Hi @SophieADav,

Great job removing the merge conflict markers! Well done!

You're super close. The only thing is that you accidentally deleted some links to other student's advice files from the index.html file.  (You can see the diff of your pull request [here](https://github.com/skillcrush/git-practice/pull/1085/files) if you are curious.)

Once you add the links back, please add the change, commit as per usual, and push up your advice branch once more, which will automatically update this pull request.

Then we should be all set to merge in your code. Let us know if you have any questions!

Best,
Lisa

**Comment by SophieADav** [comment]
Hi Lisa,

Okay, I think I fixed it again haha! Hopefully I got it right, let me know!
Thanks for all your help :)

Thanks,
Sophie

**Comment by twosavoie** [comment]
Hi @SophieADav,

You're doing great! And you're very close!

Currently, I don't see the links below in the index.html file on your advice branch. You can copy and paste them into the bottom of the file. If you do see them locally, check to be sure the changes to the index.html file have been saved. And then add and commit the changes and push to GitHub. 👍

That's it! Once that's done, your pull request should be all set to be merged into Skillcrush's repo.

Let us know if you have any questions!

Best,
Lisa

      <a href ="advice/Aminah-P-advice.html">Aminah P's advice</a><br>
      <a href="advice/jerry-weber-advice.html">Jerry Weber's Advice</a><br>
      <a href="advice/matthew-gault-advice.html">Matthew Gault's Advice</a><br>
      <a href="advice/wesley-jabs-advice.html" style="color: #300865; font-size: 20px; font-weight: bold; font-variant: small-caps;">Wesley's Wise Words of Encouragment</a><br>
     <a href="advice/patricia-reich-advice.html" target="_blank">Patricia Reich's Advice</a><br>
     <a href="advice/emily-helms-advice.html" target="_blank">Emily Helms's Advice</a><br>

**Comment by twosavoie** [comment]
Hey @SophieADav,

We haven't heard from you in a while, so we're going to go ahead and close this pull request. Feel free to submit a new one when you're ready!

Best,
Lisa

**Comment by SophieADav** [comment]
Hi Lisa,

Sorry been going through a hard time with my family at the moment, I will
be back at it soon.

Thanks for all the support!

Sophie

**Pull request closed without merging**